src/api/java/com/minecolonies/api/compatibility/ICompatibilityManager.java

```diff
@@ -172,4 +172,11 @@
      * @return the lucky ore.
      */
     ItemStack getRandomLuckyOre();
+
+    /**
+     * Get a random enchantment for a certain building level.
+     * @param buildingLevel the building level.
+     * @return a tuple of enchantment + level.
+     */
+    Tuple<String, Integer> getRandomEnchantment(final int buildingLevel);
```

Comment:
See implementation for comment on this Method.
In General different behaviour would be needed here, to make reuse more easy.